posts/2021/08/not-a-checklist.md

```diff
@@ -0,0 +1,124 @@
+can approach this for JupyterLab long-term, especially when the people we want 
+feedback from the most likely can’t use JupyterLab at all as it is now. Here 
+are a few approaches we’ve been applying so far:
+
+- Actively inviting people who haven’t been served by JupyterLab in the past, 
+disabled people in this case, back to the community. This has been mostly in 
+our [regular accessibility meetings](https://jupyter.readthedocs.io/en/latest/community/content-community.html#jupyter-community-meetings) 
+and in planning upcoming Jupyter accessibility workshops.
+- Prioritizing that the resources we use when doing accessibility work are 
+made by disabled people and/or are resources that received review from that 
+community. This also includes emphasizing a search for multiple resources and 
+approaches to similar topics rather than stopping only at one, like reading 
+[WCAG guidelines](https://www.w3.org/TR/WCAG21/) and nothing else, for example.
+- Manually testing ourselves even though we have a majority abled community. 
+[I’ve said it before](https://labs.quansight.org/blog/2021/03/accessibility-whos-responsible/), 
+but even abled users can test things, such as keyboard accessibility or 
+contrast, pretty reliably. It is important that even as we want disabled 
+people to lead, we can’t put all the pressure on them alone. Accessibility is 
+the responsibility of abled people too.
+
+Additionally, there are two approaches we’d like to be using but are currently 
+not.
+
+- Utilizing services that connect us with disabled people who work in 
+accessibility testing. We are not doing this at the moment, because we have 
+already been made aware by disabled community members that 
+[JupyterLab has major common accessibility issues](https://github.com/jupyterlab/jupyterlab/issues/9399). 
+We’d rather do this approach when we are ready to test our solutions to these 
+problems and can make the most of that feedback.
+- Utilizing automated accessibility testing as a part of JupyterLab’s standard 
+test suite. This approach is currently blocked by available tests. I know that 
+automated testing is contentious in accessibility spaces, but for reasons I 
+mentioned in [this past post](https://labs.quansight.org/blog/2021/05/putting-out-the-fire/) 
+I think it is an important part (not whole) of sustainable accessibility for 
+open source projects specifically.
+
+### What’s next?
+
+After several months of JupyterLab accessibility meetings and lovely comments 
+from people across open source communities (like our friends in [NumPy](https://numpy.org/) 
+and [Spyder](https://www.spyder-ide.org/)), people have repeatedly asked for 
+what my process is for evaluating accessibility in my regular work as an abled 
+person. Stay tuned for my personal recommendations and references for making 
+accessibility a constant consideration even when it is not a part of your 
+personal experience!
```

Comment:
Now that the cat is out of the bag we can also add a note about the CZI grant and link to the previous post in the blog?